app/services/aspace/client.rb

```diff
@@ -16,10 +16,12 @@ def initialize
       login
     end
 
-    def ead_urls
+    def ead_urls(modified_since: nil)
       output = recent_repositories.each_with_object({}) do |repository, acc|
         config.base_repo = repository["uri"][1..-1]
-        resource_ids = get("resources", query: { all_ids: true }).parsed
+        query = { all_ids: true }
+        query[:modified_since] = modified_since if modified_since
```

Comment:
I am not finding the `modified_since` parameter listed in the API operation for https://archivesspace.github.io/archivesspace/api/?shell#get-a-list-of-resources-for-a-repository